build.gradle

```diff
@@ -137,12 +137,23 @@ project('shared:protocol') {
     }
 }
 
+// Use the osdetector-gradle-plugin
+apply plugin: "com.google.osdetector"
+
+
 project('client') {
     dependencies {
         compile group: 'com.google.guava', name: 'guava', version: guavaVersion
         compile group: 'org.projectlombok', name: 'lombok', version: lombokVersion
         compile group: 'com.google.code.findbugs', name: 'annotations', version: findbugsVersion
         compile group: 'io.netty', name: 'netty-all', version: nettyVersion
+        compile "io.grpc:grpc-netty:" + grpcVersion
+        compile "io.grpc:grpc-all:" + grpcVersion
+        compile "io.grpc:grpc-protobuf:" + grpcVersion
+        compile "io.grpc:grpc-stub:" + grpcVersion
+        // https://mvnrepository.com/artifact/io.netty/netty-tcnative-boringssl-static
+        compile group: 'io.netty', name: 'netty-tcnative-boringssl-static', version: '2.0.6.Final'
```

Comment:
In one of the earlier reviews, you had similar comment. I tried using the dynamic version. I dropped it again because it was difficult to get setup on my laptop (and on my ubuntu box). That's the reason I reverted to boringssl. Will create an issue to use the dynamic version.